x-pack/test_serverless/functional/test_suites/observability/dataset_quality/degraded_field_flyout.ts

```diff
@@ -194,7 +194,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
         // Set Limit of 42
         await PageObjects.datasetQuality.setDataStreamSettings(nginxAccessDataStreamName, {
-          'mapping.total_fields.limit': 43,
+          'mapping.total_fields.limit': 42,
```

Comment:
This fails locally as well as on CI.

![image](https://github.com/user-attachments/assets/f916629b-75b1-4aea-b699-61386301bd25)

Reply:
Thanks, I ran it locally before and it passed, I'm not sure what happened. Fixed now.

Reply:
@awahab07 Could you review again? Thanks!